### PR TITLE
Batch query

### PR DESCRIPTION
**Pull request by bingyanglin**
Batch query by cdrs

### CODE REVIEW

File: src/storage.rs [hunk 1]
```diff
@@ -173,62 +173,64 @@ impl StorageBackend for CQLSession {
...
-
+        let batch = BatchQueryBuilder::new()
+            .add_query(INSERT_TX_QUERY, values)
+            .add_query(INSERT_EDGE_QUERY, bundle)
+            .add_query(INSERT_EDGE_QUERY, address)
+            .add_query(INSERT_EDGE_QUERY, tag)
+            .add_query(INSERT_EDGE_QUERY, approvee1)
+            .add_query(INSERT_EDGE_QUERY, approvee2)
+            .finalize()
+            .expect("batch builder");
```
Comment (wusyong) [inline src/storage.rs]: Return errors instead of panic here.

File: src/storage.rs [hunk 2]
```diff
@@ -173,62 +173,64 @@ impl StorageBackend for CQLSession {
...
-        // TODO: Batch query instead, but it seems curretn btach executor is not compatible to scylla.
-        self.0.query_with_values(INSERT_TX_QUERY, values)?;
-        self.0.query_with_values(INSERT_EDGE_QUERY, bundle)?;
-        self.0.query_with_values(INSERT_EDGE_QUERY, address)?;
-        self.0.query_with_values(INSERT_EDGE_QUERY, tag)?;
-        self.0.query_with_values(INSERT_EDGE_QUERY, approvee1)?;
-        self.0.query_with_values(INSERT_EDGE_QUERY, approvee2)?;
-
+        let batch = BatchQueryBuilder::new()
+            .add_query(INSERT_TX_QUERY, values)
```
Comment (wusyong) [inline src/storage.rs]: So could you explain why add_query_prepare not working here?

File: src/storage.rs (hunk 1 above)
Comment (bingyanglin) [reply src/storage.rs]: Thank you pointing this out. Modified.

File: src/storage.rs (hunk 2 above)
Comment (bingyanglin) [reply src/storage.rs]: The add_query_prepare actually works. The previous errors happen due to the macro query_value!(). In the batch query, we should not provide the queries w/ value names. See descriptions in https://github.com/apache/cassandra/blob/trunk/doc/native_protocol_v4.spec#L416-L422

**Pull request merged**